pkg/collector/process.go

```diff
@@ -69,14 +69,6 @@ func InitCollectingProcess(address net.Addr, maxBufferSize uint16, templateTTL u
 	return collectProc, nil
 }
 
-func (cp *collectingProcess) Start() {
-	if cp.address.Network() == "tcp" {
-		cp.startTCPServer()
-	} else if cp.address.Network() == "udp" {
-		cp.startUDPServer()
-	}
-}
-
```

Comment:
Is this removed to let the user start regular tcp/udp server or server with TLS/DTLS encryption? If so, I feel it is better to take in a input from the user through InitCollectorProcess on whether to use encryption or not.

Reply:
Just checking if these comments are resolved or not. From the latest code changes, it doesn't seem so.

Reply:
Thanks for reminding. Changes got postponed by other PRs. Will update asap.